docs/getting_started/building_blocks.md

```diff
@@ -392,6 +343,44 @@ This demo loads in a single object, and then dynamically modifies its state so t
     --8<-- "src/examples/object_states/object_state_texture_demo.py"
     ```
 
+### **Onfire Demo**
+!!! abstract "This demo is useful for..."
+
+    * Understanding how changing onfire state can cause objects' visual changes
+    * Understanding how onfire can be triggered by nearby onfire objects
+
+```{.python .annotate}
+python -m omnigibson.examples.object_states.onfire_demo
+```
+
+This demo loads in a stove (toggled on) and two apples. The first apple will be ignited by the stove first, then the second apple will be ignited by the first apple.
+
+??? code "onfire_demo.py"
+
+    ``` py linenums="1"
+    --8<-- "src/examples/object_states/onfire_demo.py"
+    ```
+
+### **Overlaid Demo**
+!!! abstract "This demo is useful for..."
+
+    * Understanding how cloth objects can be overlaid on rigid objects
+    * Understanding current heuristics used for gauging a cloth's "foldness"
```

Comment:
I don't think we measure folded in this demo

Reply:
done